### PR TITLE
refactor [liberapay] service

### PR DESCRIPTION
**Pull request by chris48s**

### CODE REVIEW

File: services/liberapay/liberapay-goal.service.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        namedParams: { entity: 'Changaco' },
+        staticPreview: this.render({ percentAcheived: 33 }),
+      },
+    ]
+  }
+
+  static render({ percentAcheived }) {
+    return {
+      label: 'goal progress',
+      message: `${percentAcheived}%`,
```
Comment (calebcartwright) [inline services/liberapay/liberapay-goal.service.js]: Would decimal precision ever be a consideration here? Not sure whether the underlying data points could ever allow for an ugly decimal here or not

File: services/liberapay/liberapay-goal.tester.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const Joi = require('joi')
+const t = (module.exports = require('..').createServiceTester())
+
+t.create('Goal Progress (valid)')
+  .get('/Liberapay.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'goal progress',
+      value: Joi.string().regex(/^[0-9]+%/),
```
Comment (calebcartwright) [inline services/liberapay/liberapay-goal.tester.js]: Could we use `isPercentage` from test-validators here?

File: services/liberapay/liberapay-goal.service.js (hunk 1 above)
Comment (chris48s) [reply services/liberapay/liberapay-goal.service.js]: We're already rounding it to the nearest int in `handle()`

Comment (calebcartwright) [reply services/liberapay/liberapay-goal.service.js]: My bad! Missed that

**Pull request merged**